pkg/operators/formatters/formatters.go

```diff
@@ -18,8 +18,11 @@ import (
 	"encoding/binary"
 	"fmt"
 	"net"
+	"syscall"
 	"time"
 
+	"golang.org/x/sys/unix"
```

Comment:
Just a note: we should later on think about splitting formatters into ones that could run on the client and ones that should run on the server; this dependency makes it OS dependent (I think), so would be suited for the server running operator.